serde_arrow/src/internal/serialization/date64_builder.rs

```diff
@@ -57,11 +60,33 @@ impl SimpleSerializer for Date64Builder {
     fn serialize_str(&mut self, v: &str) -> Result<()> {
         let timestamp = if self.utc {
             use chrono::{DateTime, Utc};
-            v.parse::<DateTime<Utc>>()?.timestamp_millis()
+            let date_time = v.parse::<DateTime<Utc>>()?;
```

Comment:
The after the parse the step is the same, isn't it? Could you move it out of the if / else to make that clearer?

Reply:
Good point. Fixed.